front/app/components/IdeaCards/EmptyIdeas.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors, fontSizes, defaultCardStyle } from 'utils/styleUtils';
+import { Icon } from '@citizenlab/cl2-component-library';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+const EmptyContainer = styled.div`
+  flex: 1;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  ${defaultCardStyle};
+`;
+
+const EmptyContainerInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-top: 100px;
+  padding-bottom: 100px;
+`;
+
+const IdeaIcon = styled(Icon)`
+  flex: 0 0 30px;
+  width: 30px;
+  height: 30px;
+  fill: ${colors.label};
+`;
+
+const EmptyMessage = styled.div`
```

Comment:
Should this be a `p` or a `heading` of some sort? Div doesn't feel very semantic if the component is used to display text.